source/particle/world.cc

```diff
@@ -21,12 +21,8 @@
 #include <aspect/particle/world.h>
 #include <aspect/global.h>
 #include <aspect/utilities.h>
-#include <aspect/compat.h>
-#include <aspect/geometry_model/box.h>
-#include <aspect/geometry_model/two_merged_boxes.h>
-#include <aspect/geometry_model/spherical_shell.h>
-#include <aspect/particle/integrator/euler.h>
 #include <aspect/citation_info.h>
+#include <aspect/geometry_model/interface.h>
```

Comment:
I don't think you need this any more. If it compiled before, it should be compiling now even without this addition given that you are only removing code in this file.